### PR TITLE
updates for frontend wallet connection 

### LINKED ISSUE #394: [Frontend] Change library for connection to wallets
🚨 Attention: Please read the guidelines before applying; otherwise, you will be ignored. 🚫📋
0. Please, join the telegram group to keep in touch https://t.me/+m0IGRiBvLtsyMzIy

1. Read guideline [here](https://gist.github.com/djeck1432/f722f5fe518c80b248ff609627da25e5)
2. At the moment we are using `get-starknet` but we want to change it to [`starknetkit`](https://www.starknetkit.com/)
3. Rework connection to the wallet logic [here](https://github.com/djeck1432/spotnet/blob/main/frontend/src/services/wallet.js) and [here](https://github.com/djeck1432/spotnet/blob/main/frontend/src/services/transaction.js) and [here](https://github.com/djeck1432/spotnet/blob/main/frontend/src/services/contract.js)
4. Rework also test cases for these files to make sure that github pipeline will be passed

### PR DESCRIPTION
**Pull request by Nityam573**
Closes #394

### CODE REVIEW

**Comment by iamnovichek** [comment]
@Nityam573 make sure all pipelines are green

**Comment by Nityam573** [comment]
@iamnovichek  all pipelines are green please check

**Comment by iamnovichek** [comment]
@Nityam573 What is the issue you are assigned to? I cannot find it.

**Comment by Nityam573** [comment]
@iamnovichek I wasn't assigned to any but was able to resolve this one

**Comment by iamnovichek** [comment]
@Nityam573 Did you talk about that with @djeck1432 ?

File: frontend/src/services/wallet.js [hunk 1]
```diff
@@ -15,12 +15,19 @@ export const checkForCRMToken = async (walletAddress) => {
   }
 
   try {
-    const starknet = await connect();
-    if (!starknet.isConnected) {
+    const { wallet } = await connect({
+      // connectors: [
```
Comment (iamnovichek) [inline frontend/src/services/wallet.js:19]: Do we need this commented code here?

**Review by iamnovichek — CHANGES_REQUESTED** [review]
GJ

File: frontend/src/services/wallet.js (hunk 1 above)
Comment (Nityam573) [reply frontend/src/services/wallet.js:19]: Nope not needed

**Comment by Nityam573** [comment]
@iamnovichek review please

File: frontend/src/services/wallet.js (hunk 1 above)
Comment (Nityam573) [reply frontend/src/services/wallet.js]: I removed it

**Pull request merged**